### PR TITLE
Ab#81865 assign request to mme

### PR DESCRIPTION
**Pull request by AlexandreSanscartier**

### CODE REVIEW

File: src/CSF.SRDashboard.Client/Shared/LoginDisplay.razor.cs [hunk 1]
```diff
@@ -21,6 +21,7 @@ protected override async void OnInitialized()
             {
                 this.userDisplayName = graphApiService.GetUserDisplayName();
                 this.photoDataString = graphApiService.GetUserPhotoData();
+                var groupMembers = graphApiService.GetMmeGroupMembers();
```
Comment (TCPrimedPaul) [inline src/CSF.SRDashboard.Client/Shared/LoginDisplay.razor.cs:24]: All references to MMEs would need to be changed to represent Marine Medical Staff. MMEs are the external doctors examining patients, we do not want to create confusion in code base (see discussion in PBI).

File: src/CSF.SRDashboard.Client/Services/WorkLoadManagementService.cs [hunk 2]
```diff
@@ -258,19 +269,24 @@ public WorkItemDTO UpdateWorkItemForRequestModel(RequestModel requestModel, IGat
...
             workItem.CreatedDateUTC = existingWorkItem.CreatedDateUTC;
             workItem.ReceivedDateUTC = existingWorkItem.ReceivedDateUTC;
             workItem.LastUpdatedDateUTC = DateTime.UtcNow;
             workItem.SameApplicantSubmitterInd = true;
             workItem.LineOfBusinessId = Constants.MarineMedical;
+
+            //Assignment 
+            //NOTE: Current implementation of Work load management service does not support updating 
+            // this needs to be brought up to them. Instead use the this.UpdateAssignment(WorkItemAssignmentDTO assignment, bool isToDelete)
+            workItem.WorkItemAssignment = this.GetAssignmentFromRequestModel(requestModel);
```
Comment (TCPrimedPaul) [inline src/CSF.SRDashboard.Client/Services/WorkLoadManagementService.cs:283]: Was this brought to their attention? Can we now remove this from comments?

File: src/CSF.SRDashboard.Client/Pages/SeafarerProfile.razor.cs [hunk 3]
```diff
@@ -183,7 +186,28 @@ private void LoadData()
...
             this.TableItems = WorkLoadService.GetByCdnInRequestTableFormat(Cdn);
+            this.SetAssigneeNames(this.TableItems);
             this.AlertType = AlertTypes.Success;
         }
+
+        //TODO:If an MME leaves his job and on Azure MME list does not have this person, when loading a work request
+        //where this MME was assigned, following method can not show this person.
+        //Discus this issue. Possible fix could be to store MME name on Work load management Assignee table togethe with
+        //Or instead of the AssigneeId. Or we should not delete the person fromo the MME list when this person leaves his duty.
+        private void SetAssigneeNames(List<WorkloadRequestTableItem> tableItems)
```
Comment (TCPrimedPaul) [inline src/CSF.SRDashboard.Client/Pages/SeafarerProfile.razor.cs:197]: I think I saw a discussion in PBI related to this. It seems Azure AD users are soft-deleted for 30 days; after that we would not be able to retrieve their information.

**Review by TCPrimedPaul — COMMENTED** [review]
Great work Mansuer! Some comments need to be approved before merging. We can discuss.

File: src/CSF.SRDashboard.Client/Shared/LoginDisplay.razor.cs (hunk 1 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Shared/LoginDisplay.razor.cs:24]: Good point.  Changed method name.

File: src/CSF.SRDashboard.Client/Pages/SeafarerProfile.razor.cs (hunk 3 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Pages/SeafarerProfile.razor.cs:197]: This issue was discussed DB architect Justin. I was told that Justin brought this issue to Jon's attention. There is a PBI created to address this issue. Removed the comment from the code.

File: src/CSF.SRDashboard.Client/Services/WorkLoadManagementService.cs (hunk 2 above)
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Services/WorkLoadManagementService.cs:283]: Similar issue was brought to them in the past. Had a discussion with Justin Hay. We will bring this to Krakens later again.
Removed the comment from the code.

**Pull request merged**